spec/models/evernote_note_spec.rb

```diff
@@ -37,33 +34,18 @@
   describe '#dirtify marks it dirty' do
     before { @evernote_note.dirtify }
     its(:dirty) { should == true }
-    its(:attempts) { should == 0 }
   end
 
   describe '#undirtify marks it not dirty' do
     before do
-      @evernote_note = FactoryGirl.create(:evernote_note, dirty: true, attempts: 1)
+      @evernote_note = FactoryGirl.create(:evernote_note)
       @evernote_note.undirtify
     end
     its(:dirty) { is_expected.to eq(false) }
-    its(:attempts) { is_expected.to eq(0) }
-  end
-
-  describe '#increment_attempts increments attempts' do
-    before do
-      @evernote_note = FactoryGirl.create(:evernote_note, attempts: 0)
-      @evernote_note.increment_attempts
-    end
-    its(:attempts) { is_expected.to eq(1) }
-  end
-
-  describe '#max_out_attempts increments attempts' do
-    before { @evernote_note.max_out_attempts }
-    its(:attempts) { should >=  ENV['attempts'].to_i }
   end
 
   describe '#add_task' do
-    ENV['evernote_notebooks'] = 'MYNOTEBOOK,OTHERNOTEBOOK'
+    before { ENV['evernote_notebooks'] = 'MYNOTEBOOK,OTHERNOTEBOOK' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.